src/hazelweb/Logger.re

```diff
@@ -1,6 +1,8 @@
 module Js = Js_of_ocaml.Js;
```

Comment:
this logger is for a different purpose than debugging, maybe put this stuff in a Debug module?